core/performance_test/benchmark/Cabana_NeighborArborXPerformance.cpp

```diff
@@ -0,0 +1,196 @@
+/****************************************************************************
+ * Copyright (c) 2018-2021 by the Cabana authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "Cabana_BenchmarkTimer.hpp"
+
+#include <Cabana_Core.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <numeric>
+#include <sstream>
+#include <string>
+#include <vector>
+
+//---------------------------------------------------------------------------//
+// Performance test.
+template <class Device>
+void performanceTest( std::ostream& stream, const std::string& test_prefix )
+{
+    using exec_space = typename Device::execution_space;
+    using memory_space = typename Device::memory_space;
+
+    // Declare the neighbor list type.
+    using ListTag = Cabana::FullNeighborTag;
+    using IterTag = Cabana::SerialOpTag;
+
+    // Declare problem sizes.
+    double min_dist = 1.0;
+    std::vector<int> problem_sizes = { 1000, 10000, 100000, 1000000 };
+    int num_problem_size = problem_sizes.size();
+    std::vector<double> x_min( num_problem_size );
+    std::vector<double> x_max( num_problem_size );
+
+    // Declare the number of cutoff ratios (directly related to neighbors per
+    // atom) to generate.
+    std::vector<double> cutoff_ratios = { 4.0, 6.0 };
```

Comment:
Why 4 and 6 instead of 4 and 8 as in the linked cell list benchmark?

Reply:
I meant to change everything to 4 & 6. 8 is a huge number of neighbors (from an MD point of view)